src/components/Counts/_counts.scss

```diff
@@ -0,0 +1,5 @@
+.entity-counts {
```

Comment:
These styles are necessary to position the `small` in the correct location as per the design.